### ISSUE TITLE
Reduce error logging in vertx proxy address forwarding

### ISSUE THREAD

**Issue by bobbyphilip** [quarkusio/quarkus#12667]
I see a error log when an ipv6 address of the condensed format  `2001:db8:85a3:8d3:1319:8a2e:370::` is used. (Basically ends in a ::)
> Failed to parse a port from forwarded-type headers

https://github.com/quarkusio/quarkus/blob/9d5f82255ee8a994bcffd16c0cbcc24edd6ab226/extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/ForwardedParser.java#L204

Would a change like https://github.com/bobbyphilip/quarkus/commit/98a52d8b1135470b5ee7bf582fa7316707453a6e, be acceptable to reduce the needless error log?

**Comment by gsmet** [comment]
@bobbyphilip yeah I think your proposal would be a good first step (except for the comment I added). Could you prepare a pull request? I would like to have it in 1.9.0.Final so if you prepare the PR today, that would be perfect.
